patterns/multisig/MultisigRune.ts

```diff
@@ -1,6 +1,7 @@
 import { MultiAddress } from "@capi/polkadot"
 import * as bytes from "../../deps/std/bytes.ts"
 import { $, Chain, ChainRune, PatternRune, Rune, RunicArgs, ValueRune } from "../../mod.ts"
+import * as util from "../../util/mod.ts"
```

Comment:
Can we `import { bytes }` instead?

Reply:
We are already importing bytes from std

Reply:
Im going to just merge it with std i think that is more preferrable